docs/en/trusted-solutions/tier1/features/media_replication.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Media Replication
+pagename: tier1-features-media-replication
+lang: en
+---
+
+# Media Replication
+
+Shotgun is compatible with the S3 Cross-Region replication feature, allowing your users located in different regions to read from the region closer to them in order to reduce latency and increase throughput. Replication to one region is currently supported.
+
+<img alt="media-replication-overview" src="../images/media-replication-overview.png" width="400">
+
+## Pre-requisites
+Media Isolation is required in order to elect Media Replication.
+
+## Configuration by users
+When using media replication, each users can customize from which region data is read from. A user can either specify the region to use, or use the automatic mode. In automatic mode, a user will use the replica if it is amongst the IP range specified in the Isolation Preferences.
```

Comment:
When using Media Replication, each user can customize which region data is read from. A user can either specify the region to use, or use automatic mode. In automatic mode Shotgun selects the replica determined by the user's IP address using IP ranges specified in the Isolation Preferences.